pkg/datapath/iptables/iptables.go

```diff
@@ -492,8 +492,8 @@ func (m *IptablesManager) SupportsOriginalSourceAddr() bool {
 	return (m.haveSocketMatch || m.ipEarlyDemuxDisabled) && option.Config.Tunnel == option.TunnelDisabled
 }
 
-// removeOldRules removes iptables rules installed by Cilium.
-func (m *IptablesManager) removeOldRules(quiet bool) {
+// removeOldRulesAndIpsets removes iptables rules and ipsets installed by Cilium.
+func (m *IptablesManager) removeOldRulesAndIpsets(prefix string, quiet bool) {
```

Comment:
Since this function has a `prefix` parameter, maybe it can be renamed to `removeRulesAndIpsets`? (And perhaps we can use a common prefix for both iptables and ipsets and pass the same prefix to `m.removeCiliumRules()` rather than using the `oldCiliumPrefix` constant?)

Reply:
Done and, since it was starting to look a bit too much like refactoring, I split it into a refactoring commit and the commit with changes.